src/main/kotlin/de/nycode/bankobot/twitch/TwitchIntegration.kt

```diff
@@ -95,13 +99,24 @@ private suspend fun Kord.subscribe(userId: Int, token: TwitchAccessTokenResponse
  * @param token the app authentication token
  */
 @OptIn(ObsoleteCoroutinesApi::class, ExperimentalTime::class)
+@Suppress("MagicNumber")
 private fun CoroutineScope.launchSubscriptionUpdater(
     userId: Int,
     token: TwitchAccessTokenResponse
 ) = launch {
-    for (unit in ticker(1.days.toLongMilliseconds())) {
-        BankoBot.httpClient.updateSubscription(userId, "subscribe", token)
-        println("Updated subscription!")
+    val duration = if (Config.ENVIRONMENT == Environment.PRODUCTION) {
+        1.days.toLongMilliseconds()
+    } else {
+        30.seconds.toLongMilliseconds()
+    }
+    for (unit in ticker(delayMillis = duration, initialDelayMillis = duration)) {
```

Comment:
```suggestion
    val duration = if (Config.ENVIRONMENT == Environment.PRODUCTION) {
        1.days
    } else {
        30.seconds
    }
    val delay = duration.toLongMilliseconds()
    for (unit in ticker(delayMillis = delay , initialDelayMillis = delay)) {
```


Probably do something like this I don't like the two calls to toLongMilliseconds

Reply:
👍